### ISSUE TITLE
Job for cgrulesengd.service failed

### ISSUE THREAD

**Issue by nfhost** [EvernodeXRPL/evernode-host#85]
A start job for unit cgrulesengd.service has finished with a failure.

The job identifier is 616 and the job result is failed.
Apr 08 19:19:05 systemd[1]: cgrulesengd.service: Scheduled restart job, restart counter is at 5.
Subject: Automatic restarting of a unit has been scheduled
Defined-By: systemd

 systemd[1]: cgrulesengd.service: Start request repeated too quickly.

[installer-1680980721.log](https://github.com/EvernodeXRPL/evernode-host/files/11183982/installer-1680980721.log)

**Comment by wojake** [comment]
```
STAGE Configuring host xrpl account
Using registry: *************************
Preparing host account:**********a (domain:*********** registry:********************)
STAGE Registering host on Evernode registry ************************
Invalid args.
Evernode xrpl message board exiting with error.
REG_FAILURE
Rolling back sashimono installation.
```
This seems to be the problem, did you use a funded account on the V3 testnet?

**Comment by nfhost** [comment]
Yes. 
Should I use a new one?
I thought cgrulesengd.service is the problem.

**Comment by chalith** [comment]
@nfhost Can you run `systemctl status cgrulesengd.service` command and check whether the output shows active? And can you run `journalctl -xeu cgrulesengd.service | tail -n 50` and paste the output here.

**Comment by nfhost** [comment]
After systemctl status cgrulesengd.service command:

> × cgrulesengd.service - cgroups rules generator
     Loaded: loaded (/etc/systemd/system/cgrulesengd.service; enabled; vendor preset: enabled)
     Active: failed (Result: exit-code) since Sun 2023-04-09 21:17:12 UTC; 10h ago
    Process: 14188 ExecStart=/usr/sbin/cgrulesengd (code=exited, status=1/FAILURE)
        CPU: 2ms
Apr 09 21:17:12 systemd[1]: Failed to start cgroups rules generator.
Apr 09 21:17:12 systemd[1]: cgrulesengd.service: Scheduled restart job, restart counter is at 9.
Apr 09 21:17:12 systemd[1]: Stopped cgroups rules generator.
Apr 09 21:17:12 systemd[1]: cgrulesengd.service: Start request repeated too quickly.
Apr 09 21:17:12 systemd[1]: cgrulesengd.service: Failed with result 'exit-code'.
Apr 09 21:17:12 systemd[1]: Failed to start cgroups rules generator.

journalctl -xeu cgrulesengd.service | tail -n 50 result

> ░░ Defined-By: systemd
░░ Support: http://www.ubuntu.com/support
░░
░░ An ExecStart= process belonging to unit cgrulesengd.service has exited.
░░
░░ The process' exit code is 'exited' and its exit status is 1.
Apr 09 21:17:12 vps-9e1ea102 systemd[1]: cgrulesengd.service: Failed with result 'exit-code'.
░░ Subject: Unit failed
░░ Defined-By: systemd
░░ Support: http://www.ubuntu.com/support
░░
░░ The unit cgrulesengd.service has entered the 'failed' state with result 'exit-code'.
Apr 09 21:17:12 vps-9e1ea102 systemd[1]: Failed to start cgroups rules generator.
░░ Subject: A start job for unit cgrulesengd.service has failed
░░ Defined-By: systemd
░░ Support: http://www.ubuntu.com/support
░░
░░ A start job for unit cgrulesengd.service has finished with a failure.
░░
░░ The job identifier is 10961 and the job result is failed.
Apr 09 21:17:12 vps-9e1ea102 systemd[1]: cgrulesengd.service: Scheduled restart job, restart counter is at 9.
░░ Subject: Automatic restarting of a unit has been scheduled
░░ Defined-By: systemd
░░ Support: http://www.ubuntu.com/support
░░
░░ Automatic restarting of the unit cgrulesengd.service has been scheduled, as the result for
░░ the configured Restart= setting for the unit.
Apr 09 21:17:12 vps-9e1ea102 systemd[1]: Stopped cgroups rules generator.
░░ Subject: A stop job for unit cgrulesengd.service has finished
░░ Defined-By: systemd
░░ Support: http://www.ubuntu.com/support
░░
░░ A stop job for unit cgrulesengd.service has finished.
░░
░░ The job identifier is 11051 and the job result is done.
Apr 09 21:17:12 vps-9e1ea102 systemd[1]: cgrulesengd.service: Start request repeated too quickly.
Apr 09 21:17:12 vps-9e1ea102 systemd[1]: cgrulesengd.service: Failed with result 'exit-code'.
░░ Subject: Unit failed
░░ Defined-By: systemd
░░ Support: http://www.ubuntu.com/support
░░
░░ The unit cgrulesengd.service has entered the 'failed' state with result 'exit-code'.
Apr 09 21:17:12 vps-9e1ea102 systemd[1]: Failed to start cgroups rules generator.
░░ Subject: A start job for unit cgrulesengd.service has failed
░░ Defined-By: systemd
░░ Support: http://www.ubuntu.com/support
░░
░░ A start job for unit cgrulesengd.service has finished with a failure.
░░
░░ The job identifier is 11051 and the job result is failed.

**Comment by chalith** [comment]
> my-domain

You are installing your existing ssl certificates right? Is there a dns record for your domain? Is your host ip is reachable by your domain?

**Comment by nfhost** [comment]
yes, I changed it only here ;) my domain is already encrypted with Let's encrypt https://evernode.nfhost.pl

**Comment by chalith** [comment]
@nfhost our system requirement is Ubuntu 20.04 as mentioned in https://github.com/EvernodeXRPL/evernode-host#system-requirements

**Comment by nfhost** [comment]
yup, working fine on 20.04
